README.adoc

```diff
@@ -125,7 +125,7 @@ This guide uses Docker to run an instance of the PostgreSQL database for a fast
 
 [role='command']
 ```
-docker build -t postgres-sample .
+docker build -t postgres-sample.
```

Comment:
hi @ramkumar-k-9286 

This line is command instead of a statement. The space is needed.